shippable.yml

```diff
@@ -9,24 +9,51 @@ matrix:
     - env: T=none
   include:
     - env: T=sanity/1
+    - env: T=sanity/1 A_REV=2.9
+    - env: T=sanity/1 A_REV=2.10.0a1
```

Comment:
Are you sure you want to test against ansible 2.10 instead of ansible-base 2.10? Doesn't this collection already exist in ansible 2.10?